draft/2024-02-21-this-week-in-rust.md

```diff
@@ -35,6 +35,8 @@ and just ask the editors to select the category.
 
 ### Project/Tooling Updates
 
+- [vscode-rustup](https://github.com/emberian/vscode-rustup) is released, with a status bar item to view/set workspace overrides and periodic background checks for new toolchains, so you don't miss any of those delicious delicious updates.
```

Comment:
hi @emberian 👋! we've been trying to be consistent with formatting of github repo links when submitted via the community. if you'd like to change the copy here, I'd be happy to accept a PR before the emails go out tomorrow evening :)

```suggestion
- [vscode-rustup released: interface with rustup as a VSCode extension](https://github.com/emberian/vscode-rustup)
```